map-storage/src/Services/S3Client.ts

```diff
@@ -0,0 +1,38 @@
+import { S3, S3ClientConfig } from "@aws-sdk/client-s3";
+import {
+    AWS_ACCESS_KEY_ID,
+    AWS_BUCKET,
+    AWS_DEFAULT_REGION,
+    AWS_SECRET_ACCESS_KEY,
+    AWS_URL,
+    S3_UPLOAD_CONCURRENCY_LIMIT,
+} from "../Enum/EnvironmentVariable";
+import pLimit from "p-limit";
+
+let s3: S3 | undefined;
+
+export function getS3Client(): S3 {
+    if (s3) {
+        return s3;
+    }
+
+    const config: S3ClientConfig = {
+        // ID and Secret are detected by default by the lib
+        /*accessKeyId: AWS_ACCESS_KEY_ID,
+        secretAccessKey: AWS_SECRET_ACCESS_KEY,*/
```

Comment:
Normal that this code is commented?

If it's useless, delete it

Reply:
Done